dashboard/agent.py

```diff
@@ -92,10 +86,83 @@ async def _check_parent():
                     dashboard_consts.
                     DASHBOARD_AGENT_CHECK_PARENT_INTERVAL_SECONDS)
 
+        check_parent_task = asyncio.create_task(_check_parent())
+
+        # Create an aioredis client for all modules.
```

Comment:
Why not just sharing one?

Reply:
The `aioredis_client` is shared by all the modules.